tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_types.ts

```diff
@@ -487,6 +487,19 @@ export interface DebuggerState {
   runs: DebuggerRunListing;
   runsLoaded: LoadState;
 
+  // Timestamp for the last data polling (including the
+  // initial data-loading event triggered by plugin loading,
+  // and those triggered by TensorBoard's core auto and manual
+  // reloading).
+  lastDataPollTime: number;
```

Comment:
Can you codified the contract? -1 means not loaded yet and this value can be equal to that of `lastNewPollDataTime`.

Reply:
Added the contract to the doc string here.